src/netlifyGraph.ts

```diff
@@ -206,57 +206,61 @@ ${out(
   netlifyGraphConfig,
   ["node"],
   `const httpFetch = (siteId, options) => {
-      const reqBody = options.body || null
-      const userHeaders = options.headers || {}
-      const headers = {
-        ...userHeaders,
-        'Content-Type': 'application/json',
-        'Content-Length': reqBody.length,
-      }
+  const reqBody = options.body || null;
+  const userHeaders = options.headers || {};
+  const headers = {
+    ...userHeaders,
+    "Content-Type": "application/json",
+    "Content-Length": reqBody.length,
+  };
 
-      const timeoutMs = 30_000
+  const timeoutMs = 30_000;
 
-      const reqOptions = {
-        method: 'POST',
-        headers: headers,
-        timeout: timeoutMs,
-      }
+  const reqOptions = {
+    method: "POST",
+    headers: headers,
+    timeout: timeoutMs,
+  };
 
-  const url = 'https://graph.netlify.com/graphql?app_id=' + siteId
+  const url = "https://graph.netlify.com/graphql?app_id=" + siteId;
 
-  const respBody = []
+  const respBody = [];
 
   return new Promise((resolve, reject) => {
     const req = https.request(url, reqOptions, (res) => {
       if (res.statusCode && (res.statusCode < 200 || res.statusCode > 299)) {
         return reject(
           new Error(
-            "Netlify Graph return non-OK HTTP status code" + res.statusCode,
-          ),
-        )
+            "Netlify Graph return non-OK HTTP status code" + res.statusCode
+          )
+        );
       }
 
-      res.on('data', (chunk) => respBody.push(chunk))
+      res.on("data", (chunk) => respBody.push(chunk));
 
-      res.on('end', () => {
-        const resString = buffer.Buffer.concat(respBody).toString()
-        resolve(resString)
-      })
-    })
+      res.on("end", () => {
+        const resString = buffer.Buffer.concat(respBody).toString();
+        resolve({
+          status: res.statusCode,
+          body: resString,
+          headers: res.headers,
+        });
+      });
+    });
 
-    req.on('error', (error) => {
-      console.error('Error making request to Netlify Graph:', error)
-    })
+    req.on("error", (error) => {
+      console.error("Error making request to Netlify Graph:", error);
+    });
 
-    req.on('timeout', () => {
-      req.destroy()
-      reject(new Error('Request to Netlify Graph timed out'))
-    })
+    req.on("timeout", () => {
```

Comment:
Single/double quotes should probably be standardized throughout the generated code